### DISCUSSION TITLE
Welcome to Sublime-Text-harbour-Package Discussions!

### DISCUSSION THREAD

**Discussion by asistex** [asistex/Sublime-Text-harbour-Package#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jvtecheto** [comment]
Hi 
I'm Jose Vte. From Spain and i am programmer Harbour, my interest is nowadays mod_harbour(Harbour for the web), Thanks to asistex for update the package for mod_harbour, i would like extend de syntax and functions etc. for TWeb and Mercury (Carles Aubia) for work with mod_harbour.

**Comment by AntonioFS-1962** [comment]
Hello.
My name is Antonio F.S. From Spain I discovered Harbour not long ago. So far I have investigated several code editors for Linux (Ubuntu->Mint) and with Sublime I feel very comfortable programming. I hope that updates for Harbour will be regular and I encourage its creators to continue with the project. As a qualitative suggestion: extend it to IDE rad? :-) :-)

Thank you very much.
Best regards.

Antonio F.S.
